Nav BridgeIT/css/style.css

```diff
@@ -0,0 +1,92 @@
+*{
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+}
+
+body{
+  font-size: 62.5%;
+}
+
+.flexWrapper {
```

Comment:
Nie ma sensu dodawać do nazwy klasy słowa "flex". jeżeli wrapper miałby być mieć display: flex nie musi to być aż tak eksponowane.